### ISSUE TITLE
Are you using drfpasswordless in any production apps? Would love to hear more.

### ISSUE THREAD

**Issue by aaronn** [aaronn/django-rest-framework-passwordless#68]
Would love to hear more from anyone using drfpasswordless in production. Feel free to respond here or hit me up via email.

**Comment by io-ma** [comment]
On it, will let you know how it goes!
Thank you very much for this package.

**Comment by dubesar** [comment]
Yes i am trying to use, but dont have an option for logout and login again, i think so, or i havent seen how to log back in once logged out or get the auth token

**Comment by jaymes15** [comment]
@aaronn 
yes.
Native messenger on Playstore.
![Screenshot_2022-02-28-13-51-05-286_com android vending](https://user-images.githubusercontent.com/46538679/155986539-c7b6675c-67b5-4725-8a41-ba724abc42f0.jpg)

**Comment by DarkcoderSe** [comment]
Yes, using drfpasswordless for a dating application on iOS and Android.
